docs/acme-divergences.md

```diff
@@ -46,6 +46,8 @@ Boulder does not implement the `meta` field returned by the `directory` endpoint
 
 Boulder does not implement the `terms-of-service-agreed` or `orders` fields in the registration object (nor the endpoints the latter links to).
 
+Boulder does not return the `status` field.
```

Comment:
6.1.2 defines an account resources with "status (required, string)". That's why I have chosen this section.

But imho what should be returned on POSTing is not covered by the standard. I tried to explain this on the [ACME list](https://mailarchive.ietf.org/arch/msg/acme/JrvPm8RAbSq4TqZUNAI9EvqmNys). But I am pretty sure it should be an account resource (since it has not other use in the standard).

I don't mind the exact placement ;)